### PR TITLE
feat(frontend): allow clicking 'Enter' to send passwords

### PR DESCRIPTION
**Pull request by rakeshdas1**
Added a keypress handler to the initial password input field so that a user can simply press `Enter` to get into their dashboard instead of clicking on the "Login" button.

### CODE REVIEW

**Comment by MichaIng** [comment]
Awesome, I just through the same when hitting enter today and nothing happened 😄.

**Comment by ravenclaw900** [comment]
Hi,
Thank you for this, definitely something useful to have. However, instead of using an `on:keypress` handler, why not make the textbox and button a `<form>`,  which will do the same thing, and then you should be able to replace both the `on:click` and `on:keypress` with an `on:submit`.

**Comment by rakeshdas1** [comment]
Great idea! I will implement that and remove the extraneous code.

File: src/frontend/src/App.svelte [hunk 1]
```diff
@@ -280,16 +280,19 @@
...
-                <input
-                    type="password"
-                    class="outline-none bg-gray-100 border border-gray-400 dark:border-gray-700 rounded focus:bg-gray-200 dark:bg-gray-900 dark:focus:bg-gray-800"
-                    bind:value={password}
-                />
-                <button
-                    on:click={getToken}
-                    class="border-gray-500 hover:bg-gray-100 dark:hover:bg-gray-900 focus:outline-none border p-2 rounded active:bg-gray-200 dark:active:bg-gray-800"
-                    >Login</button
+                <form
```
Comment (ravenclaw900) [inline src/frontend/src/App.svelte:283]: Please add the `flex flex-col gap-5 items-center` classes to the form, so it matches the rest of the dialog.

File: src/frontend/src/App.svelte [hunk 2]
```diff
@@ -280,16 +280,19 @@
...
+                    on:submit|preventDefault="{getToken}"
```
Comment (ravenclaw900) [inline src/frontend/src/App.svelte:284]: No need for the quotes around {getToken}, just a personal preference.

**Review by ravenclaw900 — APPROVED** [review]
Works great!

**Pull request merged**